src/families/ethereum/bridge/libcore.js

```diff
@@ -135,8 +136,16 @@ const getTransactionStatus = async (a, t) => {
     warnings.feeTooHigh = new FeeTooHigh();
   }
 
-  if (!errors.amount && amount.eq(0)) {
-    errors.amount = new AmountRequired();
+  if (!errors.amount) {
+    if (amount.eq(0)) {
+      errors.amount = new AmountRequired();
+    } else if (
+      !tokenAccount
+        ? amount.gt(account.balance.minus(estimatedFees))
+        : amount.gt(tokenAccount.balance) || estimatedFees.gt(a.balance)
```

Comment:
`estimatedFees.gt(a.balance)` should trigger `NotEnoughBalanceInParentAccount` instead of `NotEnoughBalance` cc @LFBarreto